src/http/server.rs

```diff
@@ -1048,6 +1048,48 @@ impl HttpServer {
     pub fn get_ws_channels(&self) -> HashMap<String, HashSet<u32>> {
         self.ws_channels.clone()
     }
+
+    /// Register multiple paths with the HTTP server using the same configuration.
+    /// Uses the security setting from HttpBindingConfig if secure is None,
+    /// forces secure binding if Some(true), or forces non-secure binding if Some(false).
+    /// All paths must be bound successfully, or none will be bound. If any path
+    /// fails to bind, all previously bound paths will be unbound before returning
+    /// the error.
+    pub fn bind_multiple_http_paths<T: Into<String>>(
+        &mut self,
+        paths: Vec<T>,
+        config: HttpBindingConfig,
+        secure: Option<bool>,
```

Comment:
I'm confused why the `secure` flag is necessary. `HttpBindingConfig` allows you to set whether the path will be a secure subdomain, so what's the point of an additional flag?

Reply:
I suppose I over-engineered it. It slipped my mind that `HttpBindingConfig` already allows you to declare whether a path will be secure or not. I think I remember a time where that config object didn't exist (or maybe I was unaware of it) so it didn't occur naturally. In any case, I'll get rid of it and update the PR asap.